### ISSUE TITLE
where is the "test" directory? Should I create it?

### ISSUE THREAD

**Issue by HCHCXY** [WestlakeIntelligentRobotics/ConsensusLLM-code#10]
![image](https://github.com/WestlakeIntelligentRobotics/ConsensusLLM-code/assets/64967515/e9ea3fb9-d8fa-41ae-a442-6711d7020b0c)

**Comment by huabench** [comment]
Yes, you need to create a 'test' directory and write your test files according to the instructions provided in the README.

https://github.com/WestlakeIntelligentRobotics/ConsensusLLM-code/blob/master/README.md?plain=1#L92-L127

**Comment by huabench** [comment]
Resolved, closing this issue now. Feel free to reach out if you have any further questions.
